src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -739,8 +739,12 @@ namespace HVACUnitaryBypassVAV {
                             ShowContinueError(state, format("...occurs in {} \"{}\"", thisCBVAV.UnitType, thisCBVAV.Name));
                             ErrorsFound = true;
                         } else {
-                            thisCBVAV.DXCoilInletNode = state.dataDXCoils->DXCoil(thisCBVAV.DXCoolCoilIndexNum).AirInNode;
-                            thisCBVAV.DXCoilOutletNode = state.dataDXCoils->DXCoil(thisCBVAV.DXCoolCoilIndexNum).AirOutNode;
+                            // the "coil" nodes are actually the parent nodes of the wrapped HX
+                            thisCBVAV.DXCoilInletNode =
+                                HVACHXAssistedCoolingCoil::GetCoilInletNode(state, thisCoolCoilType, thisCBVAV.DXCoolCoilName, DXErrorsFound);
+                            thisCBVAV.DXCoilOutletNode =
+                                HVACHXAssistedCoolingCoil::GetCoilOutletNode(state, thisCoolCoilType, thisCBVAV.DXCoolCoilName, DXErrorsFound);
+                            // the DX coil holds the condenser inlet node number
```

Comment:
This was just a dumb mistake during a refactor exercise. DXCoilInletNode here refers to the parent object node, not the coil inside of the parent.

Reply:
:+1: